src/Console/Command/ObfuscateData.php

```diff
@@ -0,0 +1,305 @@
+<?php namespace Ushahidi\Console\Command;
+
+/**
+ * Ushahidi Obfuscate Data console command
+ *  - This overwrites specific data within the current database
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Illuminate\Console\Command;
+use Ushahidi\Core\Entity\PostRepository;
+use Ushahidi\Core\Entity\ContactRepository;
+use Ushahidi\Core\Entity\ConfigRepository;
+use Ushahidi\Core\Entity\UserRepository;
+use Ushahidi\App\Multisite;
+use Ohanzee\DB;
+use Faker;
+use Database;
+
+class ObfuscateData extends Command
+{
+    private $postRepository;
+    private $contactRepository;
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'db:obfuscatedata';
+
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'db:obfuscatedata';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Obfuscates selected PII and removes various config keys from database';
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        //set up repos
+        $this->contactRepository = service('repository.contact');
+        $this->postRepository = service('repository.post');
+        $this->configRepository = service('repository.config');
+        $this->userRepository = service('repository.user');
+
+        $this->db = service('kohana.db');
+
+        if ($this->isThisAMultisiteInstall()) {
+            if (!getenv('HOST') || strlen(getenv('HOST')) < 1) {
+                $this->alert("ERROR: A host must be specified for a multisite deployment.");
+                exit;
+            }
+        }
+
+        if ($this->isTestEnvironment() || $this->isStagingEnvironment()) {
+            //confirm acknowledgements
+            $this->alert("WARNING: This script will wipe user, contacts, post author and data source data.");
+            if (!$this->confirm("Do you want to continue?")) {
+                $this->info("Request canceled.\n");
+                exit;
+            }
+            if (!$this->confirm('Do you acknowledge that this data will still contain sensitive data?')) {
+                $this->info("Request canceled.\n");
+                exit;
+            }
+
+            // Do overwriting
+            $this->db->begin();
+            $this->overwriteContacts();
+            $this->overwritePostAuthors();
+            $this->removeDataProviderValues();
+            $this->overwriteSiteConfig();
+            $this->deleteUsers();
+            $this->addAdminUser();
+            $this->db->commit();
+        } else {
+            $this->info("This script will only run on test or staging deployments.\n\n");
+            exit;
```

Comment:
I wonder if this should be exit or `return`? Does laravel do any handling after the `handle` call that gets skipped by `exit`?